plugins/guests/debian/cap/configure_networks.rb

```diff
@@ -11,6 +11,16 @@ class ConfigureNetworks
 
         def self.configure_networks(machine, networks)
           machine.communicate.tap do |comm|
+            main_interface = ''
+            comm.execute(%q!ip route | awk '$1=="default"{print $NF;exit}'!) do |type, data|
+              main_interface = data.chomp if type == :stdout
+            end
+            available_interfaces = []
+            comm.execute("ls /sys/class/net | grep -v -E '^(lo$|docker|lx[cd]|veth)'") do |type, data|
```

Comment:
somewhat related, CoreOS also uses systemd naming, maybe we want to unify it? https://github.com/mitchellh/vagrant/pull/6610/files